features/FEATURE_BLE/ble/pal/PalSecurityManager.h

```diff
@@ -976,12 +975,9 @@ class SecurityManager : private mbed::NonCopyable<SecurityManager> {
 
     /**
      * Generate local OOB data to be sent to the application which sends it to the peer.
-     *
-     * @param[in] connectionHandle Handle to identify the connection.
+     * @retval BLE_ERROR_NONE On success, else an error code indicating reason for failure
```

Comment:
`@retval` or `@return` ?

Reply:
Had a look at doxygen manual, should be @return here and elsewhere as well, as it appears it is being used incorrectly